pydoctor/astbuilder.py

```diff
@@ -17,10 +20,10 @@ class str_with_orig(str):
     """
     pass
 
-class MyTransformer(transformer.Transformer):
```

Comment:
FWIW, since the server code got deleted ages ago, MyTransformer and str_with_orig can go too.